worker/include/RTC/SeqManager.hpp

```diff
@@ -49,6 +49,8 @@ namespace RTC
 		void ClearDropped();
 
 	private:
+		// Whether at least a sequence number has been inserted.
+		bool started{ false };
```

Comment:
Can you explain how it works? I usually hate those flags specially in stuff that deals with number series that can start with any random number.

Reply:
The very first input value must become the current `maxInput` and `maxOutput`.

Before this flag:

* maxInput is initialized to `0` because it's a number.
* first call to `Insert(input)` arrives.
* `IsSeqHigherThan(input, this->maxInput)` => `IsSeqHigherThan(input, 0)` may be false, so `this-maxInput` remains the default value `0`, which is wrong.
* same for `maxOtput`